src/api/category/services/getMany.js

```diff
@@ -7,13 +7,23 @@ async function readMany(req, res) {
   try {
     const limit = +req.query.limit;
     const pageNo = +req.query.pageNo;
+    const { language } = req.headers;
+    const { search } = req.query;
+    const { notSelectedLanguages } = req;
     const filter = { deleted: false };
+    const dispatchedLanguage = { name: `name${req.headers.language}` };
```

Comment:
you have variable `language`, but here you are still using `req.headers.language`